src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,47 @@
+package lotto.domain;
+
+import java.math.BigDecimal;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lottos {
+    private static final int PRICE_PER_LOTTO = 1000;
+
+    private List<Lotto> lottos;
+
+    public Lottos(int money) {
+        int count = money / PRICE_PER_LOTTO;
+        checkValidation(count);
+        this.lottos = IntStream.range(0, count).mapToObj(i -> new Lotto(new Numbers())).collect(Collectors.toList());
+    }
+
+    private void checkValidation(int count) {
+        if (count < 1) {
+            throw new IllegalArgumentException("로또를 한 장 이상 구매하세요.");
+        }
+    }
+
+    public int getSize() {
+        return lottos.size();
+    }
+
+    public List<Lotto> getLottos() {
+        return this.lottos;
```

Comment:
컬렉션을 그대로 반환하면 변조를 할 수 있어 일급 컬렉션의 의미가 없어지는 것 같아요.
내부 요소를 수정하지 못하는 컬렉션을 반환하면 어떨까요?
```suggestion
        return Collections.unmodifiableList(lottos);
```